### PR TITLE
March 22 Release Notes

### PR DESCRIPTION
**Pull request by lbadenhop**
Thank you for your contribution to the Amplify-Central repo.

## Describe the changes

Enter a brief description of your changes to communicate to the reviewers what you changed and why.

## Deploy preview link

Please add the deploy preview link to the **specific page** that you've changed.

## Checklist for contributors

Before submitting this PR:

* Verify that all status checks have passed
* For more information on the Markdown linter rules, see [DavidAnson/markdownlint](https://github.com/DavidAnson/markdownlint)
* For first-time contributors, ensure that you have signed the [Axway CLA](https://cla.axway.com/)

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">✅</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">🔨</span> Latest commit | d6ea5f5 |
 |<span aria-hidden="true">😎</span> Deploy preview | http://PR-421.amplify-central.opendocs-builder.pcloud.axway.int |

File: content/en/docs/amplify_relnotes/20240322_amplify.md [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+| GitLab                                     | 1.0.3                    |
+| Kafka Cluster                              | DA=v1.0.2 / TA=v1.0.2    |
+
+| Supported SaaS (embedded) Agents                      |
+|--------------------------------------------|
+| AWS Embedded Service                       |
+| Apigee X Embedded Service                  |
+| GitHub Embedded Service                    |
+
+View agents from any organization:
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20240322_amplify.md:36]: Access the list of available agents from your organization.

File: content/en/docs/amplify_relnotes/20240322_amplify.md [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+
+* Go to [https://repository.axway.com/catalog?q=agents](https://repository.axway.com/catalog?q=agents).
+
+### Axway Agent SDK
+
+The following enhancements and bug fixes are available in this release.
+
+#### Enhancements for Agent SDK
+
+* **Security Patches** - The Agent SDK was enhanced to include some security patchesn.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20240322_amplify.md:50]: typo

File: content/en/docs/amplify_relnotes/20240322_amplify.md [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+
+#### Enhancements for SaaS (embedded) Agents
+
+There are no new enhancements tp SaaS embedded agents in this release.
+
+#### Bug fixes for SaaS (embedded) Agents
+
+The following bug fixes are in this release.
+
+* **Embedded Apigee X OAuth Discovery** - A fix was made to discover and create the Asset Request Definition for an API using an OAuth Security Scheme.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20240322_amplify.md:90]: A fix was made to discover the APIs that are using an OAuth Security Scheme. This results in Marketplace users being able to request access and generate an OAuth credential.

File: content/en/docs/amplify_relnotes/20240322_amplify.md [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+There are no known limitations in this release.
+
+### Product Foundry
+
+The following enhancements and bug fixes are available in this release.
+
+#### Feature updates for Product Foundry
+
+* **Ratings details are available from a direct url** - From an integration perspective, for instance when automating the moderation flow, the ratings details can be accessed directly using the following url format: {CentralURL}/products/details/{productLogicalName}/ratings/{ratingLogicalName}
+* **Document library document** - it is now possible to Edit any document from the Document Library by using the ellipsis menu `Edit` or using the Edit button on the Document details screen. The document type cannot be change, only the content can.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20240322_amplify.md:145]: You can only change the content, but not allowed to change the document type.

**Review by georgiana-caras — CHANGES_REQUESTED** [review]

**Review by georgiana-caras — APPROVED** [review]

File: content/en/docs/amplify_relnotes/20240322_amplify.md (hunk 1 above)
Comment (cbordier) [reply content/en/docs/amplify_relnotes/20240322_amplify.md]: Integrated.

**Review by cbordier — APPROVED** [review]
I approve

**Pull request merged**